arbnode/batch_poster.go

```diff
@@ -562,22 +552,11 @@ var EspressoFetchTransactionErr = errors.New("failed to fetch the espresso trans
 
 // Adds a block merkle proof to an Espresso justification, providing a proof that a set of transactions
 // hashes to some light client state root.
-func (b *BatchPoster) checkEspressoValidation(
-	msg *arbostypes.MessageWithMetadata,
-) error {
+func (b *BatchPoster) checkEspressoValidation() error {
 	if b.streamer.espressoClient == nil && b.streamer.lightClientReader == nil {
 		// We are not using espresso mode since these haven't been set
 		return nil
 	}
-	// We only submit the user transactions to hotshot. Only those messages created by
```

Comment:
I deleted this code because all msg's should be sent to espresso, let me know if you think that should not be the case